runtime/integration-tests/src/cases/restricted_transfers.rs

```diff
@@ -425,22 +423,6 @@ mod eth_address {
 				curr_contract,
 				curr.val(TRANSFER),
 			));
-
-			assert_ok!(
-				pallet_liquidity_pools_gateway_queue::Pallet::<T>::process_message(
-					RawOrigin::Signed(Keyring::Alice.into()).into(),
-					MessageNonceStore::<T>::get(),
-				)
-			);
```

Comment:
Shouldn't we still process the message and check for an expected error?

Reply:
From the point of view of the restriction we're trying to check if `pallet_liquidity_pools::Pallet::<T>::transfer()` passes. If it does, then the restriction passed.

Basically, in those tests, we want to check the restriction is well configured by the runtime, not that the transfer itself is done.

The end-to-end tests are in `routers.rs`

Reply:
Thanks for elaborating on that, makes sense!